modin/pandas/test/test_series.py

```diff
@@ -1135,8 +1135,9 @@ def test_array(data):
 
 
 @pytest.mark.xfail(reason="Using pandas Series.")
-def test_between():
-    modin_series = create_test_series()
+@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
+def test_between(data):
+    modin_series = create_test_series(data)
```

Comment:
why change this in codeQL PR? and for similar ones

Reply:
I wanted to get everything passing if possible and these were legitimate things caught by CodeQL.

Reply:
I left harder ones as TODOs and created issues for them.